File: src/main/java/net/thedragonteam/armorplus/container/base/ContainerBase.java
package net.thedragonteam.armorplus.container.base;

import net.minecraft.entity.player.EntityPlayer;
import net.minecraft.inventory.Container;

/**
 * @author Sokratis Fotkatzikis - TheDragonTeam
 */
public class ContainerBase extends Container {

    @Override
    public boolean canInteractWith(EntityPlayer playerIn) {
        return true;
    }
}

File: src/main/java/net/thedragonteam/armorplus/container/base/ContainerBase.kt
/*
 * Copyright (c) TheDragonTeam 2016-2017.
 */

package net.thedragonteam.armorplus.container.base

import net.minecraft.entity.player.EntityPlayer
import net.minecraft.inventory.Container

open class ContainerBase : Container() {

    override fun canInteractWith(player: EntityPlayer): Boolean {
        return true
    }
}